Contributing.md

```diff
@@ -56,6 +56,70 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the TC for resolution and that discussion and
 compromise among committers be the default resolution mechanism.
 
+# Triaging
+
+Open issues in this repo are subjected to the triage criteria:
```

Comment:
Open issues in this repo are subject to the following triage criteria

Reply:
@samkamer - done, made that change